src/cauterize.rs

```diff
@@ -130,6 +131,30 @@ fn diagnostics_to_ranges<'a>(
     Ok(ranges)
 }
 
+/// Handles (inline) module content
+fn handle_mod_diagnostic(block: &syn::ItemMod, kind: &UnusedDiagnosticKind, ident: &str) -> Option<proc_macro2::Span> {
+    use syn::Item;
+    use UnusedDiagnosticKind::*;
+
+    block.content.iter().find_map(|(b, items)| {
```

Comment:
This gives a unused variable warning at `b`.

```suggestion
    block.content.iter().find_map(|(_b, items)| {
```